llvm/utils/UpdateTestChecks/common.py

```diff
@@ -1171,6 +1171,15 @@ def transform_line_vars(match):
         return match.group(1) + rv + match.group(match.lastindex)
 
     lines_with_def = []
+    multiple_braces_re = re.compile(r"({{+)|(}}+)")
+    def escape_braces(match_obj):
+        s = match_obj.group(0)
+        escaped = ''
+        if s[0] == '{':
+            escaped = s.replace('{', '\\{')
+        else:
+            escaped = s.replace('}', '\\}')
+        return ''.join(['{{', escaped, '}}'])
```

Comment:
```suggestion
        return re.escape(match_obj.group(0))
```
I think something like this would work? As we are actually escaping for use in a regex, the use of `re.escape` seems appropriate.

Reply:
That works well, we just need to manually add the wrapping `{{` / `}}`